test/test_facebook.py

```diff
@@ -197,5 +197,40 @@ def test_parse_signed_request_when_correct(self):
         self.assertTrue('algorithm' in result)
 
 
+class TestAPIRequest(FacebookTestCase):
+    def test_request_args_by_none(self):
+        FB_VER = 2.7
+        FB_OBJECT_ID = "1846089248954071_1870020306560965"
+        token = facebook.GraphAPI().get_app_access_token(
+            self.app_id, self.secret)
+        graph = facebook.GraphAPI(access_token=token, version=FB_VER)
+
+        # args=None
+        self.assertRaises(
+            TypeError, graph.request, FB_OBJECT_ID, args=None, post_args=None)
+
+    def test_request_args_in_dict(self):
+        FB_VER = 2.7
+        FB_OBJECT_ID = "1846089248954071_1870020306560965"
+        token = facebook.GraphAPI().get_app_access_token(
+            self.app_id, self.secret)
+        graph = facebook.GraphAPI(access_token=token, version=FB_VER)
+
+        # args=dict()
+        result = graph.request(FB_OBJECT_ID, args=dict(), post_args=None)
+        self.assertEqual(result["created_time"], "2016-12-24T05:20:55+0000")
+
+    def test_request_args_by_default(self):
+        FB_VER = 2.7
+        FB_OBJECT_ID = "1846089248954071_1870020306560965"
+        token = facebook.GraphAPI().get_app_access_token(
+            self.app_id, self.secret)
+        graph = facebook.GraphAPI(access_token=token, version=FB_VER)
+
+        # default value set to: args=dict()
```

Comment:
These inner comments aren't clear - they either just state the value of `args` in the next statement (like in the previous two methods) or they mention a default value that could change in the future without affecting the test (see my previous comments).

I think it makes sense to remove them and replace them with docstrings that explain what each test is testing.